go.mod

```diff
@@ -44,10 +43,10 @@ require (
 	github.com/pquerna/cachecontrol v0.0.0-20200819021114-67c6ae64274f // indirect
 	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/common v0.14.0
-	github.com/prometheus/prometheus v1.8.2-0.20200827201422-1195cc24e3c8
+	github.com/prometheus/prometheus v1.8.2-0.20201105135750-00f16d1ac3a4
 	github.com/rs/cors v1.7.0
 	github.com/securego/gosec/v2 v2.4.0
-	github.com/shirou/gopsutil v2.20.9+incompatible
+	github.com/shirou/gopsutil v3.20.10+incompatible
```

Comment:
@james-bebbington is there any problem if we move to v3?

Reply:
This LGTM - I can build and run tests on Mac on this branch. 

The previous build issues on darwin were fixed in https://github.com/shirou/gopsutil/pull/970 and released in v3+

Reply:
Not from running the unit tests. Do you foresee any errors with this change? This change (from [this commit](https://github.com/shirou/gopsutil/commit/32d4603d01b7ef6f7934252338236f195fb732bb#diff-651006f1a6fc2f3a5066aeee77dc63fcefd40c6b4cf0afdb77847a2a20c06f61)) was needed to fix compatibility changes with updating `golang.org/x/sys`.

Reply:
I don't foresee any issues. I think Bogdan was just referring to the issue we had previously when trying to upgrade `gopsutil` that were due to a breaking change in `golang.org/x/sys`. This is fixed in v3+